app/scripts/glossaire.sh

```diff
@@ -278,7 +278,7 @@ function updateFromSVN() {
     then
         echogreen "Extract strings on svn"
         cd $install
-        nice -20 php app/inc/mozorg.php
+        nice -20 php app/scripts/tmx_mozillaorg
```

Comment:
since you added a php shebang to tmx_mozillaorg you probably don't need to explicitely call the php interpreter if you make tmx_mozillaorg executable